### PR TITLE
Updated .gitignore & added light sensor

### PR DESCRIPTION
**Pull request by mglazenborg**

### CODE REVIEW

File: src/modules.c [hunk 1]
```diff
@@ -1,11 +1,18 @@
 #include "modules.h"
 
+//Return value of temperature sensor in degrees C
```
Comment (nedroden) [inline src/modules.c]: There should be a space between // and the rest of the comment

File: src/modules.c [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
-void read_lights()
+//Returns value of light sensor between 0 and 100
+float read_lights()
 {
-    
+	ADMUX = _BV(MUX0);
+	ADCSRA = _BV(ADSC);
+	loop_until_bit_is_clear(ADCSRA, ADSC);
+
+	float lightLevel = (float)ADCW / 1024 * 100;
```
Comment (nedroden) [inline src/modules.c]: This line and the next one can be replaced with ``return (float) ADCW / 1024 * 100;``.

File: src/modules.c [hunk 3]
```diff
@@ -1,11 +1,18 @@
 #include "modules.h"
 
+//Return value of temperature sensor in degrees C
 void read_temperature()
 {
 
 }
 
-void read_lights()
+//Returns value of light sensor between 0 and 100
```
Comment (nedroden) [inline src/modules.c]: There should be a space here as well

**Review by nedroden — CHANGES_REQUESTED** [review]
Looks good

**Comment by mglazenborg** [comment]
Applied requested changes.

**Review by nedroden — APPROVED** [review]
Thank you! Pull request merged.

**Pull request merged**